targets/TARGET_Silicon_Labs/TARGET_EFM32/PeripheralNames.h

```diff
@@ -136,6 +136,13 @@ typedef enum {
 } UARTName;
 #endif
 
+#if DEVICE_CAN
+typedef enum {
+    CAN_0 = (int)CAN0_BASE,
```

Comment:
Please guard each enum entry with the respective base address define, in line with the other definitions in this file